services/onboarding.py

```
@@ -74,7 +74,8 @@ class OnboardingService:
         {"name": "detect_systems", "label": "Detect Systems", "description": "Detecting business systems"},
         {"name": "provision_specialists", "label": "Provision Specialists", "description": "Provisioning specialists"},
         {"name": "create_workflows", "label": "Create Workflows", "description": "Creating initial workflows"},
-        {"name": "complete", "label": "Complete", "description": "Onboarding complete"}
+        {"name": "complete", "label": "Complete", "description": "Onboarding complete"},
+        {"name": "activate_agency", "label": "Activate Agency", "description": "Starting 24x7 AI agency with specialist runtimes"}
     ]
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)